README.md

```diff
@@ -61,6 +61,19 @@ To install to a different directory than /usr/local:
 
     $ PREFIX=/some/path sudo make install
 
+Debugging
+=========
+
+You can enable [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) to debug memory issues for Debug builds by setting the `ASAN` option:
```

Comment:
Probably good for another commit but we should get in the practice of namespacing CMake options. The reason is that it's common practice to include a library CMakeLists.txt into larger projects but options live a global scope. I have been using the AT_ prefix but am happy to use something else.

Reply:
Since this is only used in Debug builds and not in production, I would not worry too much about it. Do you think this can conflict with other CMake projects and their options?